Uni_life_tracker/app/src/main/res/layout/activity_add_event.xml

```diff
@@ -0,0 +1,267 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+    <!--  Toolbar  -->
+    <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/black"
+            android:orientation="horizontal"
+            android:padding="15dp"
+            android:gravity="center_vertical">
+
+        <ImageView
+                android:id="@+id/iv_back"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:onClick="ClickBack"
+                android:src="@drawable/ic_back_arrow"
+                android:contentDescription="@string/back"/>
+
+        <TextView
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:text="@string/add_school_event"
+                android:textSize="20sp"
+                android:textColor="@color/white"
+                android:fontFamily="@font/roboto_mono_bold"
+                android:layout_marginStart="15dp"
+                android:gravity="start"/>
+
+        <ImageView
+                android:id="@+id/iv_save"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:onClick="ClickSave"
+                android:src="@drawable/ic_done"
+                android:contentDescription="@string/save"/>
+    </LinearLayout>
+
+    <!-- Event Type Selection -->
+    <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:gravity="center"
+            android:padding="5dp">
+
+        <!-- Assessment -->
+        <TextView
+                android:id="@+id/tv_assessment"
```

Comment:
not needed but it would be nice to have a boarder around the icon to show that assessment has been selected

Reply:
have filled & outlined icons to differentiate